ssh_check/datadog_checks/ssh_check/ssh_check.py

```diff
@@ -76,9 +76,9 @@ def check(self, instance):
         if conf.private_key_file is not None:
             try:
                 if conf.private_key_type == 'ecdsa':
-                    private_key = paramiko.ECDSAKey.from_private_key_file(conf.private_key_file)
+                    private_key = paramiko.ECDSAKey.from_private_key_file(conf.private_key_file, password=conf.password)
```

Comment:
I think we need to document in `conf.yaml.example` in both `password` and `private_key_type` about `password` being used to decrypt the key file.